db/migrations/20180908022640_action_table.js

```diff
@@ -0,0 +1,13 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('action', function (table) {
+    table.increments() // PK 
+    table.string('note', 500) //  note
+    table.string('description', 500) // description
+    table.boolean('flag').defaultTo(false) // flag
+  })
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.dropTableIfExists('action')
+};
```

Comment:
Table creation looks good and easy to read, very good. Overall the code is clean and easy to read and understand. The only comment I have is to practice commenting code so that you will remember what you have done in the future.